src/whenVisible.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Use this function to delay execution of something until an element is visible
+ * in the viewport
+ *
+ * @param element : The html element that we want to observe;
+ * @param callback : This is fired when the marker is visible in the viewport
+ */
+export const whenVisible = (
+	element: HTMLElement,
+	callback: () => void,
+): void => {
+	const io = new IntersectionObserver(([entry]) => {
```

Comment:
It's worth thinking about the experience in older browsers (IE11 😒), even if it's just to drop support for them entirely. The polyfill for IntersectionObserver is huge and certainly not worth bundling. It's likely applications that use polyfill.io will provide, or can be made to provide, IntersectiionObserver for older browsers. Should we add something to the README saying something along these lines?